apps/meow/src/lib/storage.ts

```diff
@@ -390,17 +413,13 @@ export class MStore extends ZStore {
       return;
     }
 
-    await channelDB.load(n); // load last "n" messages to memory
-    const messages = channelDB.iterator({
-      limit: n, reverse: true
-    }).collect();
-
+    const messages = await this.getChannelFeed(channel,n);
     console.log(chalk.cyanBright(`Last ${n} messages published on ${channel}`));
     for (const m of messages) {
-      const msg = m.payload.value as types.ZChainMessage;
+      const msg = m[0];
       console.log(`${chalk.green('>')} `, {
-        ...msg,
-        message: await decode(msg.message, password)
+        msg,
+        message: m[1]
       });
```

Comment:
Could you please explain why we're doing this? `m[0]` or `m[1]` is not very verbose.

Reply:
Please check above comment.

Reply:
this is just printing part , m[0] is the whole zchainMessageObject and m[1] is the decoded message